src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -30,7 +30,13 @@ class LogicalMeter:
     normal `Receiver`s, but can also be composed to form higher-order formula streams.
 
     Example:
-        ``` python
+        ```python
+        from frequenz.channels import Receiver, Sender, Broadcast
+        from frequenz.sdk.actor import DataSourcingActor, ComponentMetricsResamplingActor
+        from frequenz.sdk.timeseries._resampling import ResamplerConfig
```

Comment:
```suggestion
        from frequenz.sdk.timeseries import ResamplerConfig
```

We should avoid doing any internal `_xxx` imports in examples.